topi/python/topi/vision/resize.py

```diff
@@ -0,0 +1,41 @@
+"""TVM operator input resize compute."""
+from __future__ import absolute_import
+import topi
+
+def resize(data, out_size, layout="NCHW", align_corners=False, mode="BILINEAR", weights=None):
+    """Perform resize operation on the data.
+
+    Parameters
+    ----------
+    inputs : tvm.Tensor
+        inputs is a 4-D tensor with shape
+        [batch, channel, in_height, in_width]
+        or  [batch, in_height, in_width, channel]
+
+    out_shape: Tuple
+        Output resolution scale to
+
+    layout: string
+        either "NCHW" or "NHWC"
+
+    align_corners: Boolean
+        To preserve the values at the corner pixels
+
+    mode: string
+        either "NN" or "BILINEAR"
+
+    weights:
```

Comment:
can you just call your bilinear_weights() function here, instead of requiring users to provide weights?

Reply:
Same as above comment.